semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/expressions/MethodRef.scala

```diff
@@ -1,12 +1,20 @@
 package io.shiftleft.semanticcpg.language.types.expressions
 
 import gremlin.scala.GremlinScala
-import io.shiftleft.codepropertygraph.generated.nodes
-import io.shiftleft.semanticcpg.language.NodeSteps
+import io.shiftleft.codepropertygraph.generated.{EdgeTypes, nodes}
+import io.shiftleft.semanticcpg.language._
 import io.shiftleft.semanticcpg.language.types.expressions.generalizations.ExpressionBase
 import io.shiftleft.semanticcpg.language.types.propertyaccessors.LineNumberAccessors
+import io.shiftleft.semanticcpg.language.types.structure.Method
 
 class MethodRef(raw: GremlinScala[nodes.MethodRef])
     extends NodeSteps[nodes.MethodRef](raw)
     with ExpressionBase[nodes.MethodRef]
-    with LineNumberAccessors[nodes.MethodRef] {}
+    with LineNumberAccessors[nodes.MethodRef] {
+
+  /**
+    * Traverse to referenced method.
+    * */
+  def referencedMethod: Method =
```

Comment:
Can we rename this to `method`?

Reply:
Sure. Done.

Reply:
Actually we cannot. This is because MethodRef is inheriting from ExpressionBase which provides a method function. Thats why i suggest we stick with referencedMethod.